src/network/connection.h

```diff
@@ -732,8 +732,7 @@ class Peer {
 		virtual void reportRTT(float rtt) {};
 
 		void RTTStatistics(float rtt,
-							std::string profiler_id="",
-							unsigned int num_samples=1000);
+							std::string profiler_id = "");
```

Comment:
With the other parameters deleted as they are profiler_id will easily fit on line 734